lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -217,6 +208,37 @@ def el8?
       def el_major_version
         return os_facts['os']['release']['major'] if el?
       end
+
+      def plugin_package(name)
+        package_names = \
+          { cockpit: 'rubygem-foreman_remote_execution-cockpit',
+            docker: 'rubygem-foreman_docker',
+            smart_proxy_dynflow_core: 'rubygem-smart_proxy_dynflow_core',
+            openscap: 'rubygem-foreman_openscap',
+            tasks: 'rubygem-foreman-tasks' }
+        if el7?
+          "tfm-#{package_names.fetch(name)}"
+        elsif el8? || debian?
+          package_names.fetch(name)
```

Comment:
The Debian bit is not correct. Debian uses `ruby-` and dashes since underscores are illegal.

So in the installer we have (check `$plugin_prefix`):
https://github.com/theforeman/puppet-foreman/blob/1734ec1774fb09fd3c4f82646cd54deb3651d14d/manifests/params.pp#L88-L104

Then we transform it here:
https://github.com/theforeman/puppet-foreman/blob/1734ec1774fb09fd3c4f82646cd54deb3651d14d/manifests/plugin.pp#L25
https://github.com/theforeman/puppet-foreman/blob/1734ec1774fb09fd3c4f82646cd54deb3651d14d/manifests/plugin.pp#L33-L40